src/Games/NexusMods.Games.BethesdaGameStudios/PluginAnalyzer.cs

```diff
@@ -1,30 +1,109 @@
 using System.Runtime.CompilerServices;
+using JetBrains.Annotations;
+using Microsoft.Extensions.Logging;
+using Mutagen.Bethesda;
 using Mutagen.Bethesda.Plugins;
+using Mutagen.Bethesda.Plugins.Binary.Streams;
+using Mutagen.Bethesda.Plugins.Meta;
+using Mutagen.Bethesda.Plugins.Records;
 using Mutagen.Bethesda.Skyrim;
 using NexusMods.DataModel.Abstractions;
 using NexusMods.DataModel.Abstractions.Ids;
 using NexusMods.FileExtractor.FileSignatures;
+using NexusMods.Paths;
 using NexusMods.Paths.Extensions;
+using Noggog;
 
 namespace NexusMods.Games.BethesdaGameStudios;
 
+[PublicAPI]
```

Comment:
What is the logic for this? Isn't this in the BethesdaGameStudios project?

Reply:
Right, force of habit. This should be `UsedImplicitly`.